backend/tests/test_models.py

```diff
@@ -115,12 +116,13 @@ async def test_book_illustrations(book, base64_png):
 # Title CRUD
 ######
 @pytest.mark.asyncio
-async def test_title_create(clear_titles):
+async def test_title_create():
     ident = "wikipedia_bm_test"
     title = await Title.objects.create(
         ident=ident,
     )
     assert title.ident == ident
+    await title.delete()
```

Comment:
Why are you doing this? Using a fixture, we are sure it is going to be executed, regardless of the passing or not of the test. If an exception arise, the rest of the test is not executed and your deletion code is never called.